shared/docker/app/php-fpm-logging.conf

```diff
@@ -1,2 +1,2 @@
 access.format = '{"trace_id":"%{HTTP_X_TRACE_ID}e","time_iso8601":"%{%Y-%m-%dT%H:%M:%S%z}T","time_local":"%{%d/%b/%Y:%H:%M:%S %z}T","remote_addr":"%R","real_ip":"%{HTTP_X_REAL_IP}e","real_forwarded_for":"%{HTTP_X_FORWARDED_FOR}e","real_forwarded_proto":"%{HTTP_X_FORWARDED_PROTO}e","request_time":"%d","request_uri":"%{REQUEST_SCHEME}e://%{HTTP_HOST}e%{REQUEST_URI}e","status":"%s","request_method":"%m","http_referrer":"%{HTTP_REFERER}e","http_user_agent":"%{HTTP_USER_AGENT}e","bytes_sent":"%l","http_host":"%{HTTP_HOST}e","server_name":"%{SERVER_NAME}e","server_port":"%{SERVER_PORT}e"}'
```

Comment:
I'd like to change the name of this file. At some point, it also took on config for the ping.path, which it didn't originally have in it, meaning it's no longer just php-fpm-logging config. Would it make sense to rename this to something more generic? Alternatively, keep the logging config in one file and add another file for ping config?

Reply:
OK, now renamed to more generic app-php.conf. (to be in line with app-php.ini). We could in future seperate ping config and logging config although that might add complexity. Seems harmless to have various config in 1 file now it has generic name?

Reply:
Now renamed back as this turns out to be a reserved name and renaming it would break everything.